packages/react-scripts/template-typescript/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Some notes: We would need to keep this in sync with the auto-generated tsconfig; and these comments will trigger a git diff if the user edit any property and rerun the server

Reply:
yeah, maybe a plain file would be better.